WebGPUDemo/README.md

```diff
@@ -1,25 +1,15 @@
 # WebGPU Example
 
-Install Swift 6.2 development snapshot and Swift SDK for Wasm by following
-https://www.swift.org/documentation/articles/wasm-getting-started.html.
+Install Swift 6.2 (or later version) and a corresponding Swift SDK for Wasm by following
+["Getting Started with Swift SDKs for WebAssembly" article](https://www.swift.org/documentation/articles/wasm-getting-started.html).
 
 Build with the installed Swift SDK using JavaScriptKit's `PackageToJSPlugin` plugin. Make sure to update
-the Swift SDK in the following command to the version that matches your installed swift.org toolchain.
+the Swift SDK in the following command to the version that matches your installed Swift toolchain.
 ```
-swift package --swift-sdk swift-6.2-DEVELOPMENT-SNAPSHOT-2025-06-17-a_wasm js --use-cdn
+swift package --swift-sdk swift-6.2-RELEASE_wasm js --use-cdn
 ```
 
-If you'd like to produce a smaller binary (under 400 kB), you'll have to use
```

Comment:
@kateinoigakukun removing this due to https://github.com/swiftwasm/JavaScriptKit/commit/04c9e5e0f3ec9c4f52e81647a6b0020536f7b9eb (`SuspendingClock` and `ContinuousClock` are unavailable in Embedded Swift)